### ISSUE TITLE
Standardize on single Admin username

### ISSUE THREAD

**Issue by conorsch** [freedomofpress/securedrop#1796]
# Feature request

## Description

The Ansible config used to set up the servers at install time contains a var called `ssh_users`, which should contain the username of the Admin user account on the remote servers, created during the OS install. The username will be configured with passwordless sudo, and whitelisted for SSH access in the pam config.

To minimize error-prone typing on the command-line by Admins, it would be great to reuse the remote username throughout the site config. We haven't been reusing it to date because _technically_ Admins could specify that var as a list, which makes it impossible to choose the correct username. Given the improvements to validation logic in #1782, we should:

* enforce that the `ssh_users` var contains a single variable name
* update all documentation to remove `-u <username>` from Ansible commands
* ensure that playbooks detect a YAML list and fail with an informative message, stating the requirement to update the var locally

The trade-off for standardizing on string-over-list is that _some_ Admins may have declared the `ssh_users` var to be a list. It's impossible to state the percentage of Admins doing so accurately, since we don't have access to site-specific vars for production instances, but it's a safe bet to say that most Admins are _not_ declaring the var as a list. In fact, none of our documentation includes examples of doing so. 

The advantage for standardizing on string-over-list is that we can simply the documented commands and streamline the config process. Lack of the `-u <username>` option is a common stumbling block that generates support requests, as well, so it seems like a ripe target for improvement.

## User Stories

As a new Admin, I want to declare the username for SSH once and only once during configuration of the Admin workstation.

As an existing Admin, I want subsequent playbook runs to inform me of changes I must make to my local vars file, if the vars cannot be understood without prompting.

**Comment by msheiny** [comment]
Ohhhhh i like that `user stories` is part of the issue template now :)

This is a really solid written issue as-is - I dont have anything to add and agree with the implementation and testing high-level strategies. I would just ensure that #1781 gets in before this is in PR state so that the appropriate changes are also made to the `securedrop-admin` script (which provides a flag to provide a username on the cli). I'm not 100% we should kill that option... actually yeah i cant think of a good reason to leave it in.

**Comment by conorsch** [comment]
Good, glad we agree on the strategy here. There are a few places username consolidation should take place:

* [ ] `securedrop-admin` script (#1781)
* [ ] dynamic inventory script (#1779)
* [ ] `validate` role logic (to provide an informative error message about transition)
* [ ] throughout Admin documentation (to remove the `-u <username>` flag from Ansible commands)

Any other targets should be added to this list.

**Comment by freddymartinez9** [comment]
Conor is it your understanding that by implementing this issue, we would close #1175 as "won't fix" ? It might be _technically_ possible to implement both but that requires changes to the `.torrc` and some expanded scoping of this issue.

**Comment by conorsch** [comment]
@freddymartinez9 Not at all! We still should do a better job _documenting_ the proper procedures for multiple Admins. Even standardizing on a single username for the remote account only applies per-Admin workstation: existing instances with multiple Admin usernames present on the SecureDrop servers will simply need to update their respective Admin Workstations to declare the specific account they use.

Going forward, we can consider recommending using pubkeys to authorize/deauthorize Admins. I'd also like a "rotate SSH HidServAuth" playbook (#1742) to aid in handling staff churn among Admins.

Longer-term we'll need to reassess the assumption that each org will have a single Admin, and that Admins and Journalists are often physically copresent (in the same geographic site). Both those assumptions have caused problems in the past, and any substantial architecture changes to SecureDrop should respond accordingly.
